eslint.config.js

```diff
@@ -0,0 +1,28 @@
+import js from '@eslint/js'
+import globals from 'globals'
+import reactHooks from 'eslint-plugin-react-hooks'
+import reactRefresh from 'eslint-plugin-react-refresh'
```

Comment:
**issue**: The import for `typescript-eslint` should be `@typescript-eslint/eslint-plugin` and `@typescript-eslint/parser` for proper ESLint configuration.